GovGen_Townhalls/Meeting5.md

```diff
@@ -30,6 +30,11 @@ https://allinbits.com/blog/shape-the-future-of-atomone-with-govgen/
     - validator accounts that do not exist in genesis distribution are funded with 25 $GOVGEN (self-stake is 1 $GOVGEN)
     - validator accounts that have balance less than 25 $GOVGEN are also topped off to fund operations
     - final genesis link will be available both at https://github.com/atomone-hub/govgen-genesis and https://github.com/atomone-hub/validator/tree/main/govgen-1 as soon as complete.
+  - Initial staking distribution: we want validators to have some delegations on block 0. For this we decided on the following numbers, which can be discussed:
+    - select accounts with balance >= 25govgen
```

Comment:
```suggestion
    - select accounts with balance > 25 $GOVGEN
```

strictly more is better, as we skip funded accounts this way (we would not otherwise)